docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/mint-nft.md

```diff
@@ -88,15 +97,27 @@ pragma solidity ^0.8.0;
 import "@iota/iscmagic/ISC.sol";
 
 contract NFTContract {
+    // Event emitted when a new NFT is minted
     event MintedNFT(bytes mintID);
 
+    /**
```

Comment:
Natspec uses `///` for single line comments

The example should look like:
```/// This is NatSpec Single Line Comment
/// @title This is a Title
/// @author Author Name
/// @dev Please use this in this way
```

`/*` is used for multiline:
 ```/**
This is NatSpec Multi Line comment
This is Line 2
And so on....
*/```